src/tty.rs

```diff
@@ -28,6 +28,9 @@ use display::OnResize;
 use config::{Config, Shell};
 use cli::Options;
 
+extern crate terminfo;
+use self::terminfo::Database;
```

Comment:
The `extern crate terminfo;` line needs to be moved into `src/lib.rs`. You'll see all the other `extern crate _;` lines there, just put it next to them. Then it should also be possible to just do `use terminfo::Database`.

Just to keep things in order, I'd move the `use _` statement up below the `use libc::_;` statement. This way we have libstd imports at the top, then extern crate imports and then our own local stuff.

Reply:
I saw that you moved the `extern crate` already while I was reviewing the code.

The `use terminfo::Database;` still needs to be moved though.

Reply:
That, and everything else, should be fixed now.